core/src/test/java/hudson/model/PluginTest.java

```diff
@@ -0,0 +1,42 @@
+package hudson.model;
+
+import hudson.PluginWrapper;
+import net.sf.json.JSONObject;
+import org.junit.Test;
+
+import java.io.*;
+import java.net.URL;
+import java.util.concurrent.ExecutionException;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.*;
```

Comment:
What does the use of Mockito here gain you? Doesn't the other (modified) test not show this is unnecessary?

Reply:
Yes, I've removed this unit test and I'm now leaving only the other integration test as sufficient.